exercises/practice/allergies/.docs/instruction.append.md

```diff
@@ -0,0 +1,4 @@
+Your task is to implement two functions
+
+- `isAllergicTo`: which takes as input the patient allergy score and the allergen under test, and returns true if the patient is allergic to the given allergen
+- `listAllergens`: which takes as input a score, and return a list of allergens the patient is allergic to, listed in increasing oder of value.
```

Comment:
Do we care about punctuation? The first bullet point doesn't end with a period, but the second one does.

Reply:
> which takes as input the patient allergy score and the allergen under test, and returns true if the patient is allergic to the given allergen

Another thing that I don't know if we care about. Sometimes the phrasing is `which takes as input X`, and sometimes (below) the phrasing is `which takes X as input`.

I don't feel strongly. The second feels more natural to me, but with the example above X is really long (_the patient allergy score and the allergen under test_), which might make it a bit harder to understand if "as input" goes at the end.

I honestly don't know.

Reply:
I don't mind using proper punctuation :) Let us add the period.

I see your point about asymmetry, I am not native so I can only try to guess, but I also agree for long `X`, having the `as input` at the end is a little strange. 

Since we are on the topic, what about the colon `:` ? When I wrote the sentence
- `functionName`: which blah blah
the colon looked a little funny, is a comma better?

Reply:
I would go with either colon or no colon (no preference), but comma seems funny to me for some reason.

Let's keep the colon, I think!

Having thought about this some more I think the asymmetry doesn't matter. They're in different files, to be read at different times. I doubt anyone will notice or care.